src/experimental/tools/mediaCapabilitiesProber/probers/mediaContentType/index.ts

```diff
@@ -49,9 +49,9 @@ export default function probeContentType(
     }
     for (let i = 0; i < contentTypes.length; i++) {
       if (!MediaSource_.isTypeSupported(contentTypes[i])) {
-        return [ProberStatus.NotSupported];
+        resolve([ProberStatus.NotSupported]);
```

Comment:
I don't know what a `"Pormise"` is, but a return should have worked.

Also you do not return after the resolve here, why?